internal/Version/version.go

```diff
@@ -0,0 +1,17 @@
+package version
+
+var (
+	// Version shows the current notation-go version, optionally with pre-release.
+	Version = "v1.0.0-alpha.3"
```

Comment:
I believe @yizha1 is targeting v0.10.0-alpha.3 per a different PR.  We need to align versions also with the roadmap release notes PR @iamsamirzon

Reply:
@dtzar  - Agree on aligning on the right versions to use. I had intentionally recommended updating the major revision to go from 0 to 1 because the new Notation client and even the specs have been significantly changes from the last release.

I suggest we use the version numbering I had recommend in https://github.com/notaryproject/roadmap/pull/64